e2e/test/scenarios/question/new.cy.spec.js

```diff
@@ -28,6 +28,8 @@ import {
   tableHeaderClick,
   onlyOnOSS,
   createQuestion,
+  openNotebook,
```

Comment:
I can't reproduce this locally and it might just be a video compression artifact but just to make sure: @iethree do these buttons (data picker trigger and column picker trigger) have the same height in your browser?
In your screencasts there seems to be 1px difference.

![image](https://github.com/user-attachments/assets/84cb283c-b2ff-46b1-98b6-7fe862cdf873)

Reply:
yep, it's 36 v 37 px. I can reproduce on stats. 

![Screen Shot 2024-08-01 at 7 15 31 AM](https://github.com/user-attachments/assets/2eabdf1c-282e-472f-8c37-99379030f62e) ![Screen Shot 2024-08-01 at 7 15 26 AM](https://github.com/user-attachments/assets/982d2569-fa8d-4a0b-a46a-82b9b5362ee0)

Reply:
Thanks for the confirmation! I reported it separately: #46408